File: D1-Morning/D1-S2-PHP/W15-ManyToMany/Student.php
<?php


class Student
{
    public $regNo;
    public $name;
    public $email;

    /**
     * Student constructor.
     * @param $name
     * @param $regNo
     * @param $email
     */
    public function __construct($regNo, $name, $email)
    {
        $this->name = $name;
        $this->regNo = $regNo;
        $this->email = $email;
    }


}

File: D1-Morning/C#/D1-S2/W15-ManyToMany/W15-ManyToMany/Student.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace W15_ManyToMany
{
    public class Student
    {
        public string RegNo { set; get; }
        public string Name { set; get; }
        public string Email { set; get; }

        public Student()
        { }

        public Student(string regNo, string name, string email)
        {
            RegNo = regNo;
            Name = name;
            Email = email;
        }
    }
}